### PR TITLE
Add connection to loading page and update the bottom one

### PR DESCRIPTION
**Pull request by amazingandyyy**

### CODE REVIEW

**Comment by amazingandyyy** [comment]
#713

**Comment by amazingandyyy** [comment]
screen shot 2018-07-09 at 16 14 26

**Comment by amazingandyyy** [comment]
screen shot 2018-07-09 at 16 16 28
screen shot 2018-07-09 at 16 16 33

File: src/scenes/App/components/Loader/index.js [hunk 1]
```diff
@@ -37,12 +36,9 @@ export default class Loader extends Component {
...
           </div>
           <div className={classes.loaderPercentWrapper}>
-            <Typography variant="display1" className={classes.loaderPercent}>{syncPercent}</Typography>%
-            <p>
-              <FormattedMessage id="str.blockSync" defaultMessage="Blockchain syncing..." />
-              &nbsp;
-              <FormattedMessage id="str.wait" defaultMessage="Please wait." />
-            </p>
+            <Typography variant="display1" className={classes.loaderPercent}>{syncPercent}</Typography>
+            <p>%</p>
```
Comment (henryzbw) [inline src/scenes/App/components/Loader/index.js]: keep `%` in the same line with `syncPercent`

Comment (amazingandyyy) [reply src/scenes/App/components/Loader/index.js]: why, <p></p> is a different component, and they will still be on the same line.

**Pull request merged**